jiant/models.py

```diff
@@ -1160,6 +1174,69 @@ def _lm_forward(self, batch, task, predict):
             pass
         return out
 
+    def _masked_lm_forward(self, batch, task, predict):
+        mlm_probability = 0.15
```

Comment:
I think hard-coded constants like this are only okay if we make it clear (in docs, method names, etc.) that we're _only_ supporting the exact RoBERTa setup. Otherwise, make this configurable?